lib/response.js

```diff
@@ -707,7 +707,10 @@ res.get = function(field){
  */
 
 res.clearCookie = function clearCookie(name, options) {
-  var opts = merge({ expires: new Date(1), path: '/' }, options);
+  // Force cookie expiration by setting expires to the past
+  const opts = Object.assign({ path: '/' } , options, { expires: new Date(1) })
```

Comment:
```suggestion
  const opts = { path: '/', expires: new Date(1), ...options };
```

Reply:
ha.. and now I see the comments about eslint not liking spread

Reply:
Its a good callout. Easier to land this functionally equivalent assign and then cleanup later once unblocked 👍